packages/react-magma-dom/src/components/StyledButton/styles.ts

```diff
@@ -104,25 +128,33 @@ export function buildButtonBackground(props) {
 export function buildBorderColor(props) {
   props = buildPropsWithDefaultButtonStyles(props);
 
-  if (props.disabled && props.isInverse && props.variant === 'outline') {
-    return props.theme.colors.disabledInverseText;
-  }
   if (props.disabled) {
-    return props.theme.colors.neutral06;
+    if (props.isInverse) {
+      if (props.color === 'secondary') {
+        return transparentize(0.8, props.theme.colors.neutral100);
+      }
+      return 'none';
+    }
+    if (props.color === 'secondary') {
+      return props.theme.colors.neutral300;
+    }
```

Comment:
Can this be removed if the color is the same as the default if it's not inverse?